build-config/resources/Ballerina.toml

```diff
@@ -8,4 +8,4 @@ keywords = ["Candid", "Nonprofit Data", "Philanthropy Data", "Nonprofit APIs"]
 repository = "https://github.com/ballerina-platform/module-ballerinax-candid"
 icon = "icon.png"
 license = ["Apache-2.0"]
-distribution = "2201.8.0"
+distribution = "2201.8.4"
```

Comment:
Do we need to specify this distribution version ? Are we using any new lang features or OpenAPI tool features released under this particular version ?

Reply:
Nope, we don't